packages/uid/src/uid.ts

```diff
@@ -36,30 +36,29 @@ const dictionaryLengthMinus1 = dictionary.length - 1
  *
  * @module uid
  * @param {String} prefix a string to prefix the id for debugging in non-production env
- * @param {Number} length id length (in characters, minus the prefix)
+ * @param {Number} length id length (in characters, minus the prefix). Default is 12
  * @returns {String} a unique id
  */
 function uid(prefix = '', length = 12) {
-  const id = `u${_uid('', length - 1)}`
+  const id = `u${_uid(length - 1)}`
   if (prefix && process.env.NODE_ENV !== 'production') {
     return `${prefix}__${id}`
   } else {
     return id
   }
 }
 
-// @ts-expect-error ts-migrate(7006) FIXME: Parameter 'size' implicitly has an 'any' type.
-function _random(size) {
+function _random(size: number) {
   const result = []
   /* eslint-disable-next-line no-param-reassign */
   while (0 < size--) {
-    result.push(Math.floor(Math.random() * 256))
+    // `| 0` is faster than `Math.floor()`.
```

Comment:
this idea is from https://github.com/ai/nanoid/blob/main/non-secure/index.js

Reply:
I found [some warnings](https://stackoverflow.com/questions/2526682/why-is-javascripts-math-floor-the-slowest-way-to-calculate-floor-in-javascript) about this method (`Math.floor()` and `n | 0` are not equivalent), but might probably work here, since we know it is a positive number, etc.